Makefile

```diff
@@ -258,6 +259,17 @@ ifeq ($(USE_CONTAINER),1)
 else
 	go generate ./internal/winmanifest
 endif
+
+generate-snmp:
+ifeq ($(USE_CONTAINER),1)
+	$(RERUN_IN_CONTAINER)
+else
+	@LATEST_SNMP_VERSION=$$(go list -m github.com/prometheus/snmp_exporter | cut -d' ' -f2); \
+	sed -i "s|snmp_exporter/[^/]*/snmp.yml|snmp_exporter/$$LATEST_SNMP_VERSION/snmp.yml|" internal/static/integrations/snmp_exporter/common/common.go; \
```

Comment:
It'd be nice to have comments above the `sed` commands to explain what they do. I'm personally not familiar with `sed`, and I suspect many others are also not.

Reply:
Thanks, updated.